core/migrations/20220807142446_add_user_session_activity_columns.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class AddUserSessionActivityColumns extends AbstractMigration
+{
+    public function change(): void
+    {
+        $table = $this->table('nl2_users_session');
```

Comment:
Can we add the "login method" column here?

Reply:
Done